src/CommunityToolkit/Xamarin.CommunityToolkit/Views/TabView/TabBadgeView.shared.cs

```
@@ -38,7 +38,11 @@ public bool AutoHide
 			set => SetValue(AutoHideProperty, value);
 		}
 
-		static async void OnAutoHideChanged(BindableObject bindable, object oldValue, object newValue) => await (bindable as TabBadgeView)?.UpdateVisibilityAsync();
+		static async void OnAutoHideChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			if (bindable is TabBadgeView tabBadgeView)
```

Can we use the cast? To throw an invalidCastException